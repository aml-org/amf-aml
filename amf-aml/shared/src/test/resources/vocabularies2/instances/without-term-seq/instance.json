{
  "$dialect": "amf-json-schema-generated-dialect 1.0",
  "inboundHeaders": [
    {
      "key": "x-requests-id",
      "value": "test"
    }
  ]
}
