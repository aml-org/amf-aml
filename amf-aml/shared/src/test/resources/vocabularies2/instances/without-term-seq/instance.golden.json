{
  "inboundHeaders": [
    {
      "key": "x-requests-id",
      "value": "test"
    }
  ]
}
